schedule/yast/autoyast_salt_formulas.yaml

```diff
@@ -2,6 +2,13 @@
 name: autoyast_salt_formulas
 description: |
   Test installation using AutoYaST plus salt formulas.
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_salt.xml
+  AUTOYAST_CONFIRM: '1'
+  AUTOYAST_PREPARE_PROFILE: '1'
+  SALT_FORMULAS_PATH: yast2/salt.tar.gz
+  DESKTOP: textmode
+  INSTALLONLY: '1'
```

Comment:
HDDSIZEGB=30 & HDDMODEL=scsi-hd for the same reason as for autoyast_keyboard_layout. but you can try to run it without to see if it is still working without